tests/ORSO/testORSOValidation.m

```diff
@@ -0,0 +1,61 @@
+classdef testORSOValidation < matlab.unittest.TestCase
+%%
+% Runs ORSO Validations.
+% (For now, we exclude test 4 and test 5, as these test resolution
+% functions which will diverge until phase 2 of the resolution calculations
+% is implemented)
+
+    properties
+        orsoTolerance = 1.0e-10;
+    end
+
+    properties (TestParameter)
+        layersFile = {'test0.layers', 'test1.layers', 'test2.layers', 'test3.layers', 'test6.layers', 'test7.layers'}
+        dataFile = {'test0.dat', 'test1.dat', 'test2.dat', 'test3.dat', 'test6.dat', 'test7.dat'}
+        % layersFile = {'test0.layers', 'test1.layers', 'test2.layers', 'test3.layers', 'test0.layers', 'test1.layers', 'test6.layers', 'test7.layers'}
```

Comment:
Not sure why the comments are here

Reply:
I wanted to make sure the correspondence between dat files and layers files was noted down in the main test. For the excluded tests they use previous layers files.